roles/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -81,3 +87,10 @@
   delay: 5
   retries: 20
   until: trimaran_test_pod_state.stdout == 'Completed'
+
+- name: Dump trimaran info
+  shell: |
+    oc get pod trimaran-test -n trimaran -oyaml > "{{ artifact_extra_logs_dir }}/trimaran-test.pod.yaml"
+    oc describe pod trimaran-test -n trimaran > "{{ artifact_extra_logs_dir }}/trimaran-test.pod.desc"
+    oc get all -n trimaran -oyaml > "{{ artifact_extra_logs_dir }}/all-trimaran.yaml"
+    cp {{ trimaran_setup_config }} "{{ artifact_extra_logs_dir }}/trimaran-templated-config.yaml"
```

Comment:
this can be removed after the suggestions above have been applied

Reply:
Do we no longer want to save this info or does it automatically get saved with the new path?

Reply:
we do save it :)
here is the way I would write the block above, which makes it clearer that it's stored in the artifacts:
```
- name: Create the src artifacts directory
  file:
    path: "{{ artifact_extra_logs_dir }}/src/"
    state: directory
    mode: '0755'

- name: Configure Trimaran options
  template:
    src: "{{ trimaran_setup_template }}"
    dest: "{{ artifact_extra_logs_dir }}/src/trimaran-resources.yaml"
```